src/client/driver/process-skip-js-errors.ts

```diff
@@ -0,0 +1,40 @@
+import { SkipJsErrorsOptions } from '../../configuration/interfaces';
+import ClientFunctionExecutor from './command-executors/client-functions/client-function-executor';
+import { isClientFunctionCommand } from '../../test-run/commands/utils';
+
+export async function shouldSkipJsError (options: SkipJsErrorsOptions | boolean, err: any): Promise<boolean> {
+    if (typeof options === 'boolean')
+        return options;
+
+    if (typeof options === 'object' && isClientFunctionCommand(options))
+        return processJsErrorsFunction(options, err);
+
+    if (typeof options === 'object')
+        return processJsErrorsOptions(options, err);
+
+    return false;
+}
+
+function processJsErrorsOptions (options: SkipJsErrorsOptions, err: any): boolean {
```

Comment:
We can expose this function and cover its logic in server tests.
It allowing reduce the number of functional tests.